### ISSUE TITLE
Error with new version and using the lg 27gr95qe-b Rolling back to 9.0.0 fixes it

### ISSUE THREAD

**Issue by DitherMan420** [Maassoft/ColorControl#200]
I get this huge long error about the name being wrong or invalid or something, and the nvidia tab disappears. If I go back to the last version, it's fixed. just an fyi. Monitor is LG 27gr95qe-b.

**Comment by Maassoft** [comment]
Can you give me an error log or screenshot of the problem?

**Comment by DitherMan420** [comment]
> Can you give me an error log or screenshot of the problem?

Yes, I will grab that later today or tomorrow. Sorry for the delay!

**Comment by DitherMan420** [comment]
> Can you give me an error log or screenshot of the problem?

Sorry man, I ended up returning this monitor. It was just a buggy piece of ****. So I can't test it further for you :-/.
